testing/steps/test_expand_dim.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+
+from neuraxle.pipeline import Pipeline
+from neuraxle.steps.flow import ExpandDim
+from neuraxle.steps.misc import HandleCallbackStep, TapeCallbackFunction
+
+
+def test_expand_dim_transform():
+    handle_fit_callback = TapeCallbackFunction()
+    handle_transform_callback = TapeCallbackFunction()
```

Comment:
I suggest editing the checkpoint benchmark example in this PR as well. You could have a unit test for that too, to test the expand dims on checkpoints in an isolated fashion to assert that just the good amount of files were created on disks and not more or less.

Reply:
@guillaume-chevalier I would love to do it in the next PR. I prefer keeping my changes small in a PR... I already have this in mind.